client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -84,7 +128,8 @@ const FileInfoCard: FunctionComponent< FileInfoCardProps > = ( { siteId, item }
 		}
 	}, [ downloadUrl, prepareDownloadStatus ] );
 
-	const showActions = item.type !== 'archive';
+	const showActions =
+		item.type !== 'archive' || ( item.type === 'archive' && item.extensionType === 'unchanged' );
```

Comment:
This means that the download button will be shown only when the item is a plugin or theme that is unchanged.
For example, clicking on `original version X.X from WordPress.org`.

<img width="327" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1488641/01df66da-34ea-46d7-ba9f-ef04ec9fd0f4">